cfme/roles.py

```diff
@@ -412,7 +420,8 @@ def _remove_from_all(roles, r_page):
             'Clouds': ['Instances'],
             'Infrastructure': ['Datastores', 'Providers', 'Virtual Machines', 'Hosts', 'Clusters',
                                'Resource Pools'],
-            'Physical Infrastructure': ['Providers', 'Racks', 'Switches', 'Servers', 'Topology']},
+            'Physical Infrastructure': ['Providers', 'Chassis', 'Racks', 'Switches', 'Servers',
```

Comment:
In 5.10 the order is Server , Storage , Switch , Topology , if the order matters here

Reply:
Ultimately it doesn't - explained in a comment above:

https://github.com/ManageIQ/integration_tests/pull/8530/files#r262910147